src/component/absensi/shift/Shift.vue

```diff
@@ -19,74 +17,93 @@
                 </tr>
             </thead>
             <tbody>
-                <tr>
-                    <td>{{}}</td>
-                    <td>{{}}</td>
-                    <td>{{}}</td>
-                    <td>{{}}</td>
-                    <td>{{}}</td>
-                    <td>{{}}</td>
-
+                <tr v-for="item in shiftItems"
+                    :key="item.id"
+                    :item="item">
```

Comment:
My bad. I thought this was Presensi page, so I said that we need `:key="item.id" :item="item"` and made a change on `Presensi.vue`. I've been fixed this on the latest commit